build.gradle

```diff
@@ -15,6 +15,7 @@ versions {
 }
 
 ext.isCiServer = System.getenv().get("CI") != null
+ext.testVersionFromLocalRepository = project.hasProperty('testVersionFromLocalRepository')
```

Comment:
Should we use some configuration-cache compatible way to get this property?

Reply:
Also, let's add some comment here describing the purpose of this parameter.

Reply:
Both things done.